src/App.js

```diff
@@ -1,15 +1,37 @@
 import React from 'react';
-import './App.css';
+import { useSelector } from 'react-redux';
 import { Routes, Route } from 'react-router-dom';
+import SplashScreen from './components/SplashScreen/SplashScreen';
 import Register from './components/Register/Register';
 import Login from './components/Login/Login';
+import Doctors from './components/Doctors/Doctors';
+import BookAppointment from './components/Appointments/BookAppointment';
+import MyAppointments from './components/Appointments/MyAppointments';
+import Navbar from './components/layouts/Navbar';
+import AddDoctor from './components/Doctors/AddDoctor';
+import DeleteDoctor from './components/Doctors/DeleteDoctor';
+import DoctorDetails from './components/Doctors/DoctorDetails';
 
 function App() {
```

Comment:
- [ ] The app is not responsive enough in small devices:

![image](https://user-images.githubusercontent.com/57421823/206867509-bb0f32e5-b9f0-47b9-8985-d9526101ae63.png)

Kindly make the app responsive, as this is a requirement for this project:

![image](https://user-images.githubusercontent.com/57421823/206867537-90929a10-7337-4b9d-9280-7de935291f1b.png)

Reply:
@Meltrust we have updated the changes requested